deployment/ansible/group_vars/development_template

```diff
@@ -25,3 +24,15 @@ use_s3_storage: false
 default_admin_username: 'admin'
 default_admin_password: 'admin'
 default_admin_email: 'systems+cac@azavea.com'
+
+# for database
+postgresql_version: 9.4
+postgresql_package_version: "9.4.19-1.pgdg14.04+1"
+
+postgis_version: 2.1
+postgis_package_version: "2.1.8+dfsg-5~97.git43a09cc.pgdg14.04+1"
```

Comment:
Despite the package name here, after provisioning the `database` virtual machine from scratch, it is installing PostGIS 2.4.4:

```sql
cac_tripplanner=# SELECT PostGIS_full_version();
                                                                              postgis_full_version
---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
 POSTGIS="2.4.4 r16526" PGSQL="94" GEOS="3.4.2-CAPI-1.8.2 r3921" PROJ="Rel. 4.8.0, 6 March 2012" GDAL="GDAL 1.10.1, released 2013/08/26" LIBXML="2.9.1" LIBJSON="0.11.99" RASTER
(1 row)
```

We may have to build the extension from source if we aren't able to get PostgreSQL to use the correct version.